staging-service-client/src/main/java/com/github/cafdataprocessing/services/staging/client/StagingApi.java

```diff
@@ -38,10 +35,34 @@
  *
  */
 public class StagingApi extends com.github.cafdataprocessing.services.staging.client.internal.StagingApi {
+    private final String USER_AGENT_HEADER_NAME = "User-Agent";
+    private final String USER_AGENT_HEADER_VALUE = "caf-staging-service-java-client/1.0.0-SNAPSHOT";
```

Comment:
Is the version number hard-coded?

Reply:
Removed. Using the default user-agent set by code generator instead